README.md

```diff
@@ -1,6 +1,17 @@
-# `check-protocols`
+# Check Protocols [![CircleCI](https://circleci.com/gh/Originate/check-protocols.svg?style=svg)](https://circleci.com/gh/Originate/check-protocols) [![Waffle.io - Columns and their card count](https://badge.waffle.io/Originate/check-protocols.svg?columns=all)](https://waffle.io/Originate/check-protocols)
+
+Test driven development for bash scripts (or any script, really)
+
+## Description
+
+Automated tests help us write well-behaved applications, and that's great, but
+what about all those pesky deploy scripts? How do we test those?
```

Comment:
The tool is not just for deploy scripts. So maybe '[...] what about all those pesky little scripts, e.g. deploy scripts? [...]'